docs/developer-guide/base-maps/using-with-mapbox.md

```diff
@@ -2,7 +2,7 @@
 
 | Pure JS | React | Overlaid | Interleaved |
 | ----- | ----- | ----- | ----- |
-|  ✓ | ✓ | [example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/maplibre) | [example](https://deck.gl/gallery/mapbox-layer) |
+|  ✓ | ✓ | [example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/pure-js/mapbox) | [example](https://deck.gl/examples/mapbox) |
```

Comment:
Hey @Pessimistress, per our discussion this morning - the website example (safegraph) is the only interleaved mapbox example now, so I had this linking to that. However, we agreed we don't point people to the website example in docs.

Maplibre's gallery example is the minimal interleaved example for the submodule, so do you agree we should make an exception with mixing maplibre/mapbox and link to that?

Reply:
I've updated the link to our interleaved overlay example.